GITFLOW.md

```diff
@@ -27,56 +27,56 @@ git commit -m "second commit"
 # rebase against staging to pull in any changes that have been made
 # since you started your feature branch.
 git fetch
-git rebase origin/develop
+git rebase origin/master
 
 # push your local changes up to the remote
 git push
 
 # if you've already pushed changes and have rebased, your history has changed
 # so you will need to force the push
 git fetch
-git rebase origin/develop
+git rebase origin/master
 git push --force-with-lease
 ````
 
 
 ### GitHub workflow
 
-- Open a Pull Request against `staging`. Check our PR guidelines [here](https://github.com/quintoandar/butterfree/blob/master/CONTRIBUTING.md#pull-request-guideline).
+- Open a Pull Request against `master`. Check our PR guidelines [here](https://github.com/quintoandar/butterfree/blob/master/CONTRIBUTING.md#pull-request-guideline).
 - When the Pull Request has been approved, merge using `squash and merge`, adding the Jira task number and a brief description:
 ie, `[MLOP-169] Enable stream pipelines in Butterfree`.
 - This squashes all your commits into a single clean commit. Remember to clean detailed descriptions, otherwise our git logs will be a mess.
 
-If you are unable to squash merge because of conflicts, you need to rebase against `staging` again:
+If you are unable to squash merge because of conflicts, you need to rebase against `master` again:
 
 ```
 # in your feature branch
 git fetch
-git rebase origin/staging
+git rebase origin/master
 # fix conflicts if they exist
 git push --force-with-lease
 ```
 
 
 ## Releases
 
-A release takes the changes in `staging` and applies them to `master`.
+A release takes the changes in `release` and applies them to `master`.
```

Comment:
There is no "release" branch. A release is made out of release tag. Actually the entire instruction section for releases is this: creating a release tag